ci/cd/mxnet_lib/mxnet_lib_pipeline.groovy

```diff
@@ -0,0 +1,169 @@
+// -*- mode: groovy -*-
+
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+// Jenkins pipeline
+// See documents at https://jenkins.io/doc/book/pipeline/jenkinsfile/
+
+// NOTE: 
+// ci_utils is loaded by the originating Jenkins job, e.g. jenkins/Jenkinsfile_release_job
+// cd_utils is loaded by the originating Jenkins job, e.g. jenkins/Jenkinsfile_release_job
+
+def get_pipeline(mxnet_variant, build_fn) {
+  return {
+    stage("${mxnet_variant}") {
+      stage('Build') {
+        timeout(time: max_time, unit: 'MINUTES') {
```

Comment:
Maybe make a small comment at the top of all global variables that are being referenced to avoid confusion.